tests/br_s3/run.sh

```diff
@@ -51,66 +51,74 @@ for i in $(seq $DB_COUNT); do
     run_sql "CREATE DATABASE $DB${i};"
     go-ycsb load mysql -P tests/$TEST_NAME/workload -p mysql.host=$TIDB_IP -p mysql.port=$TIDB_PORT -p mysql.user=root -p mysql.db=$DB${i}
 done
+S3_KEY=""
+for p in $(seq 2); do
 
-for i in $(seq $DB_COUNT); do
-    row_count_ori[${i}]=$(run_sql "SELECT COUNT(*) FROM $DB${i}.$TABLE;" | awk '/COUNT/{print $2}')
-done
+  for i in $(seq $DB_COUNT); do
+      row_count_ori[${i}]=$(run_sql "SELECT COUNT(*) FROM $DB${i}.$TABLE;" | awk '/COUNT/{print $2}')
+  done
 
-# backup full
-echo "backup start..."
-BACKUP_LOG="backup.log"
-rm -f $BACKUP_LOG
-unset BR_LOG_TO_TERM
-run_br --pd $PD_ADDR backup full -s "s3://mybucket/$DB?endpoint=http://$S3_ENDPOINT" \
-    --log-file $BACKUP_LOG || \
-    ( cat $BACKUP_LOG && BR_LOG_TO_TERM=1 && exit 1 )
-cat $BACKUP_LOG
-BR_LOG_TO_TERM=1
-
-if grep -i $MINIO_SECRET_KEY $BACKUP_LOG; then
-    echo "Secret key logged in log. Please remove them."
-    exit 1
-fi
+  # backup full
+  echo "backup start..."
+  BACKUP_LOG="backup.log"
+  rm -f $BACKUP_LOG
+  unset BR_LOG_TO_TERM
+  run_br --pd $PD_ADDR backup full -s "s3://mybucket/$DB?endpoint=http://$S3_ENDPOINT$S3_KEY" \
+      --log-file $BACKUP_LOG || \
+      ( cat $BACKUP_LOG && BR_LOG_TO_TERM=1 && exit 1 )
+  cat $BACKUP_LOG
+  BR_LOG_TO_TERM=1
 
-for i in $(seq $DB_COUNT); do
-    run_sql "DROP DATABASE $DB${i};"
-done
+  if grep -i $MINIO_SECRET_KEY $BACKUP_LOG; then
+      echo "Secret key logged in log. Please remove them."
+      exit 1
+  fi
 
-# restore full
-echo "restore start..."
-RESTORE_LOG="restore.log"
-rm -f $RESTORE_LOG
-unset BR_LOG_TO_TERM
-run_br restore full -s "s3://mybucket/$DB" --pd $PD_ADDR --s3.endpoint="http://$S3_ENDPOINT" \
-    --log-file $RESTORE_LOG || \
-    ( cat $RESTORE_LOG && BR_LOG_TO_TERM=1 && exit 1 )
-cat $RESTORE_LOG
-BR_LOG_TO_TERM=1
-
-if grep -i $MINIO_SECRET_KEY $RESTORE_LOG; then
-    echo "Secret key logged in log. Please remove them."
-    exit 1
-fi
+  for i in $(seq $DB_COUNT); do
+      run_sql "DROP DATABASE $DB${i};"
+  done
 
-for i in $(seq $DB_COUNT); do
-    row_count_new[${i}]=$(run_sql "SELECT COUNT(*) FROM $DB${i}.$TABLE;" | awk '/COUNT/{print $2}')
-done
+  # restore full
+  echo "restore start..."
+  RESTORE_LOG="restore.log"
+  rm -f $RESTORE_LOG
+  unset BR_LOG_TO_TERM
+  run_br restore full -s "s3://mybucket/$DB$S3_KEY" --pd $PD_ADDR --s3.endpoint="http://$S3_ENDPOINT" \
```

Comment:
```suggestion
  run_br restore full -s "s3://mybucket/$DB?$S3_KEY" --pd $PD_ADDR --s3.endpoint="http://$S3_ENDPOINT" \
```

the `?` is necessary! demo:

```go
	u, _ := url.Parse("s3://a/b&c=d")
	fmt.Println(u.Query()) // map[]
	u, _ = url.Parse("s3://a/b?&c=d")
	fmt.Println(u.Query()) // map[c:[d]]
```